src/yt_napari/_model_ingestor.py

```diff
@@ -38,6 +38,15 @@ def __init__(
         self.grid_width = self.width / self.resolution
 
 
+class LayerMetadata(dict):
+    # a container for standard metadata for a layer
+    def __init__(self, data: np.ndarray, layer_domain: LayerDomain, is_log: bool):
+        self.__setitem__("_data_range", (data.min(), data.max()))
```

Comment:
do you need to call `__setitem__`?  And, do you need a call to `super()`?

Reply:
both good questions -- and after further reading I think it might be better to just have a dict factory function instead. I only added this so I'd have an easy way to ensure consistent keys for the metadata dict. But since I'm not actually overriding any dict methods, it's safer to do something like 

```python
def create_metadata_dict(data, layer_domain, is_log):
      metadata = {}
      metadata["_data_range"] =
      < ------- etc -----------> 
      return metadata
```

And I thought about creating a separate non-dict container class, but napari is expecting a plain dict for the metadata argument, so I decided to keep it simpler for now. So ya, thanks for flagging this! Will change things to use a factory function instead.